### PR TITLE
docs: CSS reset

### PR DESCRIPTION
**Pull request by alexandereneroth**
**Describe pull-request**  
- [x] Adds a suggested CSS reset to the foundations part of the Storybook documentation.

**Solving issue**  
#DTS-886

**How to test**  
- [ ] Open the deploy preview and read the CSS reset page.

### CODE REVIEW

**Comment by mJarsater** [comment]
I think this is good but I don't think it should live under Foundations. I think that one is more for design foundations.

File: core/src/stories/foundations/css-reset.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+<Meta title="Foundations/CSS Reset" />
+
+# CSS Reset
+
+While our web components look and work as intended out of the box, basic HTML elements like `html`, `p`, `h1`, etc, still need some CSS tweaks to work predictably across browsers.
+
+For this reason we recommend a minimal CSS reset, the included comments explain what each part is for.
+
+The CSS Reset is based on [Joshua Comeaus reset](https://www.joshwcomeau.com/css/custom-css-reset). 
+For an in-depth explanation of each part we recommend reading [this article](https://www.joshwcomeau.com/css/custom-css-reset) on his website.
```
Comment (timrombergjakobsson) [inline core/src/stories/foundations/css-reset.stories.mdx:12]: Is it necessary to link to same article twice?

Comment (alexandereneroth) [inline core/src/stories/foundations/css-reset.stories.mdx:12]:
```suggestion
The CSS Reset is based on [Joshua Comeaus reset](https://www.joshwcomeau.com/css/custom-css-reset). 
For an in-depth explanation of each part we recommend reading the article on his website.
```

Comment (alexandereneroth) [reply core/src/stories/foundations/css-reset.stories.mdx]: Not sure, but I think I made it work now without doing that

**Comment by alexandereneroth** [comment]
> I think this is good but I don't think it should live under Foundations. I think that one is more for design foundations.

I was struggling deciding where to put it. 
I don't think it fits under any headline, maybe utilities is better? dunno

**Comment by alexandereneroth** [comment]
> I think this is good but I don't think it should live under Foundations. I think that one is more for design foundations.

Or under Intro perhaps

**Comment by timrombergjakobsson** [comment]
> > I think this is good but I don't think it should live under Foundations. I think that one is more for design foundations.
> 
> I was struggling deciding where to put it. I don't think it fits under any headline, maybe utilities is better? dunno

Yes, tough one, unsure though if it is a utility. Maybe helpers? Or we introduce a CSS one, where this one can live and maybe future css only version?

**Comment by alexandereneroth** [comment]
I guess foundations is sacred and intro is more of a "miscellaneous", so ill try put it there

**Comment by alexandereneroth** [comment]
> > > I think this is good but I don't think it should live under Foundations. I think that one is more for design foundations.
> > 
> > 
> > I was struggling deciding where to put it. I don't think it fits under any headline, maybe utilities is better? dunno
> 
> Yes, tough one, unsure though if it is a utility. Maybe helpers? Or we introduce a CSS one, where this one can live and maybe future css only version?

Hmm not sure we'll have more to put under a CSS headline though, maybe

**Comment by timrombergjakobsson** [comment]
> > > > I think this is good but I don't think it should live under Foundations. I think that one is more for design foundations.
> > > 
> > > 
> > > I was struggling deciding where to put it. I don't think it fits under any headline, maybe utilities is better? dunno
> > 
> > 
> > Yes, tough one, unsure though if it is a utility. Maybe helpers? Or we introduce a CSS one, where this one can live and maybe future css only version?
> 
> Hmm not sure we'll have more to put under a CSS headline though, maybe

I meant if we would also put the only css-version there. But maybe will not happen. So then maybe uneccessary.

File: core/src/stories/foundations/css-reset.stories.mdx
Comment (mJarsater) [inline core/src/stories/foundations/css-reset.stories.mdx:1]: File needs to be moved out of foundations.

Comment (alexandereneroth) [reply core/src/stories/foundations/css-reset.stories.mdx:1]: Nice catch, moved it

**Review by mJarsater — APPROVED** [review]
LGTM

**Pull request merged**